test/test_other.py

```diff
@@ -13381,5 +13381,8 @@ def test_memory_init_file_unsupported(self):
 
   @only_windows('This test verifies Windows batch script behavior against bug https://github.com/microsoft/terminal/issues/15212')
   def test_windows_batch_file_dp0_expansion_bug(self):
-    open('build_with_quotes.bat', 'w').write(f'@"emcc" {test_file("hello_world.c")}')
-    self.run_process(['build_with_quotes.bat'])
+    create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
+    # Emscripten must be available in PATH for this test
+    path = path_from_root() + os.pathsep + os.getenv('PATH')
+    with env_modify({'PATH': path}):
```

Comment:
If you want you could do `@with_env_modify({'PATH': path})` on the test function.  lgtm either way.

Reply:
check